spotlight-client/src/contentApi/sources/us_pa.ts

```diff
@@ -44,6 +43,7 @@ const content: TenantContent = {
   systemNarratives: {
     Prison: {
       title: "Prisons",
+      previewTitle: "Prison Population over Time, by Race or Ethnicity",
```

Comment:
we are not actually showing the race/ethnicity breakdown in the preview so this should also just say "Prison Population over Time" like it does in ND